src/patterns/filtering/multi-selection.stories.ts

```diff
@@ -0,0 +1,139 @@
+import { Component } from "@angular/core";
+import { storiesOf, moduleMetadata } from "@storybook/angular";
+import { withKnobs } from "@storybook/addon-knobs/angular";
+import { TableModule, TableModel, TableHeaderItem, TableItem } from "../../table/table.module";
+import { DropdownModule } from "../../dropdown/dropdown.module";
+import { GridModule } from "../../grid/grid.module";
+import { UIShellModule } from "../../ui-shell/ui-shell.module";
+import { filterErrorsAndWarnings } from "@angular/compiler-cli";
+
+@Component({
+    selector: "app-sample-multi-selection",
+    template: `
+    <div ibmGrid>
+        <div ibmRow class="header">
+            <ibm-header name="Patterns">
+                <ibm-hamburger></ibm-hamburger>
+            </ibm-header>
+        </div>
+        <div ibmRow>
+            <div ibmCol [columnNumbers]="{'lg': 3, 'md': 3, 'sm': 3}">
+                <label ibmText class="dropdown-label">
+                    Filter by:
+                    <ibm-dropdown
+                        class="filter-dropdown"
+                        type="multi"
+                        placeholder="Type"
+                        inline="true"
+                        (selected)="onSelected($event)">
+                        <ibm-dropdown-list [items]="items"></ibm-dropdown-list>
+                    </ibm-dropdown>
+                </label>
+            </div>
+        </div>
+        <div ibmRow>
+            <div ibmCol [columnNumbers]="{'lg': 12, 'md': 12, 'sm': 12}">
+                <ibm-table-container>
+                    <ibm-table
+                        class="data-table"
+                        [model]="model"
+                        size="lg"
+                        [showSelectionColumn]="false">
+                        <ng-content></ng-content>
+                    </ibm-table>
+                </ibm-table-container>
+            <div>
+        </div>
+    </div>
+    `,
+    styles: [`
+        .header {
+            margin-bottom: 80px;
+        }
+
+        .dropdown-label {
+            display: flex;
+            align-items: center;
+        }
+
+        .data-table {
+            width: 100%;
+        }
+        
+        .filter-dropdown {
+            margin-left: 20px;
+            flex-grow: 1;
+        }
+    `
+    ]
+})
+class SampleMultiSelection {
+    model = new TableModel();
+
+    items = [
+        { content: "Vegetable" },
+        { content: "Fruit" },
+        { content: "Meat" }
+    ];
+
+    dataset = [
+        { name: "Apple", type: ["Fruit"] },
+        { name: "Grape", type: ["Fruit"] },
+        { name: "Eggplant", type: ["Fruit"] },
+        { name: "FruitVegMeat", type: ["Fruit", "Vegetable", "Meat"] },
+        { name: "Lettuce", type: ["Vegetable"] },
+        { name: "Daikon Radish", type: ["Vegetable"] },
+        { name: "Beef", type: ["Meat"] }
+    ];
+    
+    onSelected(event) {
+        let checkboxFilters = [];
```

Comment:
Never reassigned so it should be
```suggestion
        const checkboxFilters = [];
```